### PR TITLE
Validation UI

### PR DESCRIPTION
**Pull request by victor-paunescu**

### CODE REVIEW

File: src/image-upload-service.ts [hunk 1]
```diff
@@ -34,10 +34,13 @@ export class ImageUploadService {
     const adGroups = this._googleAdsApi.getAdGroups();
     Logger.log(adGroups);
     for (const adGroup of adGroups) {
+      const imgFolder = CONFIG['Validated DIR']
```
Comment (akushnarov) [inline src/image-upload-service.ts:37]: How about a shorter notation `CONFIG['Validated DIR'] || CONFIG['Generated DIR']`?

File: src/index.ts [hunk 2]
```diff
@@ -18,8 +18,18 @@ import { menu } from './menu';
...
 import { ImageUploadService } from './image-upload-service';
+import { FRONTEND_HELPER } from './frontend-helper';
 
 menu;
 ImageExtensionService;
 ImageGenerationService;
 ImageUploadService;
+FRONTEND_HELPER;
+
+function doGet() {
```
Comment (akushnarov) [inline src/index.ts:29]: I would consider moving these two function to a separate file, like we did with the `menu` above.

File: src/frontend-helper.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  const gcsImages = gcsApi.listAllImages(CONFIG['Account ID']);
+  const adGroups: { [id: string]: Image[] } = {};
+  if (!gcsImages.items) {
+    return [];
+  }
+  gcsImages.items.forEach(e => {
+    const statusFolder = e.name.split('/')[2];
+    let status: IMAGE_STATUS | null = null;
+    switch (statusFolder) {
+      case 'GENERATED':
```
[the remark this answers is not in the document]
Comment (victor-paunescu) [reply src/frontend-helper.ts:54]: Great point! Done.

File: src/index.ts (hunk 2 above)
Comment (victor-paunescu) [reply src/index.ts]: Done.

**Review by akushnarov — APPROVED** [review]

**Pull request merged**